server/mergin/sync/workspace.py

```diff
@@ -248,16 +248,16 @@ def filter_projects(
     def workspace_count():
         return 1
 
-    def projects_query(self, name=None, workspace=None):
+    def projects_query(self, like: str = None):
```

Comment:
Is this used only in private API?

Reply:
Yeah, I found just one usage of ws_handler.projects_query ... and that's in private controller for ```list_projects``` ...